agent_plugin_builder/build_plugin.py

```diff
@@ -28,17 +26,21 @@ def main():
     if not plugin_path.exists():
         raise FileNotFoundError(f"Plugin path {plugin_path} does not exist")
 
+    build_agent_plugin(plugin_path)
+
+
+def build_agent_plugin(plugin_path: Path):
     agent_plugin_manifest = _get_agent_plugin_manifest(plugin_path)
 
-    # Temp build dir name
-    tmp_build_dir = tempfile.mkdtemp(
-        prefix=f"build_{agent_plugin_manifest.plugin_type}_{agent_plugin_manifest.name}_"
-    )
-    build_dir = Path(tmp_build_dir)
+    plugin_build_dir = Path.cwd() / "build"
```

Comment:
Can we make the build location customizable? Perhaps add an argument? Be wary of using a default argument